src/helpers/adapters/mysql.ts

```diff
@@ -186,6 +186,14 @@ export async function getFollowers() {
   return await db.queryAsync(query);
 }
 
+export async function getOneDayVotes() {
+  const query = `
+    SELECT space, count(DISTINCT voter, proposal) as count FROM votes
```

Comment:
Can you make the SQL syntax in uppercase like this:
```suggestion
    SELECT space, COUNT(DISTINCT voter, proposal) AS count FROM votes
```